sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -38,6 +38,8 @@ let client = AzureCommunicationChatClient(baseUrl: baseUrl, authPolicy: authPoli
 
 A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat. Users also get near real-time updates for when others are typing and when they have read the messages.
 
```

Comment:
Should this sentence be deleted? It reads like duplicate of newly added paragraph. 
"Users also get near real-time updates for when others are typing and when they have read the messages."